lib/forest_liana/bootstraper.rb

```diff
@@ -522,6 +539,60 @@ def stripe_integration_deprecated?
       is_deprecated
     end
 
+    def setup_mixpanel_integration(collection_name_and_field)
+      collection_name = collection_name_and_field.split('.')[0]
+      model_name = ForestLiana.name_for(collection_name.constantize)
+      # TODO: Remove once lianas prior to 2.0.0 are not supported anymore.
+      model_name_old = ForestLiana.name_old_for(collection_name.constantize)
+      collection_display_name = model_name.capitalize
+
+      fields = [
+        { field: :id, type: 'String', 'is-filterable': false },
+        { field: :event, type: 'String', 'is-filterable': false },
+        { field: :date, type: 'Date', 'is-filterable': false },
+        { field: :city, type: 'String', 'is-filterable': false },
+        { field: :region, type: 'String', 'is-filterable': false },
+        { field: :country, type: 'String', 'is-filterable': false },
+        { field: :timezone, type: 'String', 'is-filterable': false },
+        { field: :os, type: 'String', 'is-filterable': false },
+        { field: :osVersion, type: 'String', 'is-filterable': false },
+        { field: :browser, type: 'String', 'is-filterable': false },
+        { field: :browserVersion, type: 'String', 'is-filterable': false },
```

Comment:
all these fields are `'is-virtual': true, 'is-sortable': false,...` cf. the comments on the Express liana.